Tests/js/views/services/plugins/assets/ez-contentcreateplugin-tests.js

```diff
@@ -272,7 +290,6 @@ YUI.add('ez-contentcreateplugin-tests', function (Y) {
             this.service.set('location', location);
             this.service.fire('whatever:createContent', {
                 contentType: type,
-                languageCode: languageCode,
```

Comment:
given https://github.com/ezsystems/PlatformUIBundle/pull/593/files#r63476850, this should be kept but you'll have to change the assertion to check that this language code is ignored.